Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/__init__.py

```diff
@@ -95,6 +96,8 @@ def main(
         Blame, Branch, Canonicalize, Checkout,
         Clean, Clone, Find, Info, Land, Log, Pull,
         PullRequest, Revert, Review, Setup, InstallGitLFS,
+        Clean, Clone, Conflict, Find, Info, Land, Log, Pull,
+        PullRequest, Revert, Setup, InstallGitLFS,
```

Comment:
This change broke buildbots.
Bug 275137 – REGRESSION(279720@main): argparse.ArgumentError: argument {help,blame,branch,canonicalize,checkout,clean,clone,find,list,info,land,log,pull,up,update,pull-request,pr,pfr,upload,revert,review,setup,install-git-lfs}: conflicting subparser: clean
https://bugs.webkit.org/show_bug.cgi?id=275137
https://github.com/WebKit/WebKit/pull/29526